### DISCUSSION TITLE
roadmap/

### DISCUSSION THREAD

**Discussion by giscus** [imarvinle/CSGuide#8] [Announcements]
# roadmap/

计算机专业自学、Java、C++学习路线、C、Redis学习路线、MySQL学习路线、Python学习路线、如何学习计算机

https://csguide.cn/roadmap/

**Comment by wuxiaobin1995** [comment]
小北，有没有想开一个嵌入式软件开发的路径

**Reply by imarvinle** [reply]
好主意，我后面去找走嵌入式的大佬写一下，这一块我不是专业的~
